recipes/django-material/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "django-material" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "5ad516f55f05e2fabb31797d4c5ae5819ae37c8945386d947f09eb550bf4463b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+
+about:
+  home: http://forms.viewflow.io/
+  license: BSD 3-Clause
+  license_file: LICENSE.txt
```

Comment:
The problem is that the license file is called `LICENSE.txt`, but the `MANIFEST.in` points to a non-existent file called `LICENSE`, so no license file is included in the source distribution. I just put in a PR to fix the issue: https://github.com/viewflow/django-material/pull/291

For now you should just xref the PR here and stick a comment in the `meta.yaml` and then follow the instructions for packaging the license manually:

http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually

In future updates to the feedstock, assuming there is a new upstream merge and my PR gets accepted, you can remove the local copy of the license.